web/simonamarcu2/html/primul/simy.css

```diff
@@ -25,19 +25,21 @@ a {
 
 h1 {
   border-radius: 3px;
-  /* border: 4x solid rgb(20, 114, 177); */
-  display: block;
-  width: 250px;
+    /* border: 4x solid rgb(20, 114, 177); */
```

Comment:
If you know that in future, you will not use line 28, you can delete it, it's taking extra space.

Reply:
Thank you for suggestion. I kept it there because I've tried different ways to style my header.